qlib/utils/data.py

```
@@ -10,6 +10,7 @@
 import pandas as pd
 
 from qlib.data.data import DatasetProvider
+from qlib.data.data import LocalDatasetProvider
```

Ruff (0.11.9)

`qlib.data.data.LocalDatasetProvider` imported but unused

Remove unused import: `qlib.data.data.LocalDatasetProvider`

(F401)